### PR TITLE
mimic

### PR DESCRIPTION
**Pull request by auroracaffeine**

### CODE REVIEW

**Review by Topkinsme — APPROVED** [review]
I've approved and checked this myself off Github, needs one more approval

File: weak_power/Mimic [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Mimic-3
```
Comment (12Metatron) [inline weak_power/Mimic]: i think the 3 is supposed to be in brackets. other than that its good.

Comment (Topkinsme) [reply weak_power/Mimic]: I'll just do this since it's an oversight on my part

**Comment by Topkinsme** [comment]
We're also missing the list addition

**Comment by Topkinsme** [comment]
Done with #98

**Pull request merged**